mlflow/protos/databricks_uc_registry_messages.proto

```diff
@@ -71,14 +71,17 @@ message ModelVersion {
   // ID of the MLflow experiment containing the MLflow run that generated this model version
   optional string run_experiment_id = 9;
 
-  // ID of the workspace containing the MLflow run that generated this model version
-  optional int64 run_workspace_id = 10 ;
+  // Optional tracking server ID (e.g. Databricks workspace ID) of the source run
```

Comment:
In the future, could be the Databricks account ID + metastore ID of the source run, if experiments/runs are also moved into the UC